docs/guides/modernize-jest.md

```diff
@@ -119,4 +119,6 @@ To debug failing tests using Chrome Devtools or an IDE you can run the test comm
 node --inspect-brk ./node_modules/.bin/ng test libname
 ```
 
+> Note: on Windows OS try this instead: `node --inspect-brk ./node_modules/.bin/ng test libname`
```

Comment:
You can replace the above with the right path. It will work on unix and windows.

Reply:
changed, also fixed obvious mistake in the changed code. Squashed